missas/core/facades/google_maps.py

```
@@ -0,0 +1,62 @@
+
+    try:
+        gmaps = googlemaps.Client(key=settings.GOOGLE_MAPS_API_KEY)
+
+        data = gmaps.places(
+            query=search_query,
+            language="pt-BR",
+            region="br",
+            type="church",
+        )
+
+        logger.debug(f"Google Places API response for query '{search_query}': {data}")
```

Ruff (0.14.3)

Logging statement uses f-string

(G004)